CONTRIBUTING.md

```diff
@@ -74,18 +121,46 @@ var/path_type = /obj/item/weapon/baseball_bat
 var/path_type = "/obj/item/weapon/baseball_bat"
 ```
 
-* You must use tabs to indent your code, NOT SPACES.
+###Tabs not spaces
+You must use tabs to indent your code, NOT SPACES.
+
+(You may use spaces to align something, but you should tab to the block level first, then add the remaining spaces)
+
+###No Hacky code
+Hacky code, such as adding specific checks, is highly discouraged and only allowed when there is no other option.
```

Comment:
>only allowed when there is no other option.

aka when anyone faces a single little problem and can't think for a second or ask for help. Remove that